userguide/tasks.rst

```diff
@@ -1649,105 +1649,113 @@ shows the configuration options for Cloud Syncs.
+   +---------------------+---------------------+------------------------------------------------------------------------------------------------------------+
+   | Storage Class       | drop-down menu      | Classification for each S3 object. Choose a class based on the specific use case or performance            |
+   |                     |                     | requirements. See                                                                                          |
+   |                     |                     | `Amazon S3 Storage Classes <https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html>`__   |
+   |                     |                     | for more information on which storage class to choose.                                                     |
+   |                     |                     | :guilabel:`Storage Class` only appears when an S3 credential is the *Provider*.                            |
+   |                     |                     |                                                                                                            |
+   +---------------------+---------------------+------------------------------------------------------------------------------------------------------------+
+   | Folder              | browse button       | The name of the predefined folder within the selected bucket or container. Type the name or click          |
+   |                     |                     | |ui-browse| to list the remote filesystem and choose the folder.                                           |
+   |                     |                     |                                                                                                            |
+   +---------------------+---------------------+------------------------------------------------------------------------------------------------------------+
+   | Use --fast-list     | checkbox            | Only appears with a compatible :guilabel:`Credential`.                                                     |
+   |                     |                     | `Use fewer transactions in exchange for more RAM <https://rclone.org/docs/\#fast-list>`__.                 |
+   |                     |                     | This can also speed up or slow down the transfer.                                                          |
+   |                     |                     |                                                                                                            |
+   +---------------------+---------------------+------------------------------------------------------------------------------------------------------------+
+   | Encryption          | drop-down menu      | Only appears when an S3 credential is the *Provider*. Choices are *None* (no encryption) or                |
```

Comment:
Is the "the" needed?
"Only appears when an S3 credential is *Provider*."
In other words, "the Provider" sounds like it would not be literally Provider, but something else.

Reply:
Deleted it. "Encyrption" doesn't appear in the UI.